web-client/public/js/sliders.js

```diff
@@ -83,121 +40,13 @@ export var sliderGroupController = function (sliderArray) {
         }
     };
 
-    /* temporary code block
-    setSliderHandlers = function () {
-        sliders.activate();
-    };
-    this.activate = function () {
-        $(this.sliderId).on("input", {slider: this}, function (event) {
-            updateSliderDisplayedValue(event.data.slider, this);
-        });
-    };
-    var updateSliderDisplayedValue = function (slider, element) {
-        var value = $("#" + $(element).attr("id")).val();
-        $(slider.valueId).html(value + ((slider.needsAppendedZeros &&
-          (value.length === GRAVITY_LENGTH_WITHOUT_ZERO)) ? "0" : ""));
-        slider.setCurrentVal(value);
-    };
-    */
-
-    this.setCurrentVal = function (newVal) {
-        this.currentVal = newVal;
-    };
-
+// Object error when moving it
     this.initializeDefaultForces = function () {
```

Comment:
this function is triggered in graph.js, however, if I just replace `sliderController.initializeDefaultForces();` with `modifyChargeParameter(-50);` & `modifyLinkDistanceParameter(500)` then I get the following error, which I have never seen before.  A google search associates it with TypeScript (?).

<img width="1328" alt="screen shot 2018-11-29 at 2 03 53 pm" src="https://user-images.githubusercontent.com/5462077/49255002-9fdb3500-f3df-11e8-89ea-bc4b3e7af09e.png">

Reply:
The error means that something that the code is trying to treat like a function is actually an Object and not the function that the code thinks it is.

It can be easily tracked down by ascertaining the exact value that it is the expression before `(-50)` in the original code.

However, I need to see the version of the source code that triggers this error in order to explain more fully. What you're seeing here is already processed by `webpack` and is harder to debug in this form. Easier to debug in the pre-`webpack` form of the code.